Core/SiteRatingScoreExtension.swift

```diff
@@ -120,7 +120,7 @@ public extension SiteRating {
     }
 
     public func networkNameAndCategory(forDomain domain: String) -> ( networkName: String?, category: String? ) {
-        if let tracker = disconnectMeTrackers.first(where: { domain.hasSuffix($0.key) } )?.value {
+        if let tracker = disconnectMeTrackers.first(where: { domain == $0.key || domain.hasSuffix(".\($0.key)") } )?.value {
```

Comment:
Should this one be lowercased too? It be worth extracting this into a separate method as it happens in a few places.

Reply:
Thanks, probably. Will update, test and merge.